test/unit/app/data/domain/cdoTaskList.test.js

```diff
@@ -27,6 +27,7 @@ describe('CdoTaskList', () => {
   describe('Task List', () => {
     test('should show applicationPackSent in default state', () => {
       const cdoTaskList = buildDefaultTaskList()
+      cdoTaskList._cdo.dog._status = null
```

Comment:
Is this an accurate view of the state of a newly created Dog?  Wouldn't it be 'Pre-Exempt'?

Reply:
Have changed this now.